jax/numpy/lax_numpy.py

```diff
@@ -1153,6 +1153,25 @@ def ravel(a, order="C"):
   return reshape(a, (size(a),), order)
 
 
+_UNRAVEL_INDEX_DOC = """\
+Unlike numpy's implementation of unravel_index, negative indices are accepted
+and out-of-bounds indices are clipped.
+"""
+
+@_wraps(onp.unravel_index, lax_description=_UNRAVEL_INDEX_DOC)
+def unravel_index(flat_index, shape):
```

Comment:
I just noticed: the numpy implementation names the first argument ``indices``; we should match that. It also accepts an ``order`` argument that defaults to ``C``: it would be useful to add that here and raise a NotImplementedError (or implement it, if you're up for that).

Reply:
Should this be a separate issue to detect mismatch and missing arguments in `_wraps`? Will let you take care of this in either case. Thanks for the reviews.

Reply:
Just to be clear: you're saying you'd like me to take over the PR to rename this variable name?

Reply:
Added the rename and a clarifying change. Not sure I understand the barrier you are seeing to handling missing optional arguments in `_wraps`. Why would there be additional exceptions? Especially if you'll be updating the reference numpy version, might be less maintenance overall.

Reply:
I tried quickly comparing wrapped and unwrapped function signatures via ``inspect.signature``, and a large number of functions differ (e.g. using args/kwargs vs explicit names, jax not implementing deprecated keywords, jax adding extra documented keywords, etc.) My feeling is there are too many exceptions to make a test like that useful.

Reply:
We must be looking at this differently, because such a degree of mismatch sounds like a case for checking optional arguments in `_wraps`. Moved to #2988